tests/pom.xml

```diff
@@ -282,9 +312,18 @@
                                 <PORT>2379</PORT>
                             </environmentVariables>
                             <systemPropertyVariables>
-                                <server.zip>
-                                    ${basedir}/../distribution/target/wso2am-micro-gw-${project.version}.zip
-                                </server.zip>
+                                <toolkit>
+                                    ${basedir}/../tests/target/micro-gwtmp/wso2am-micro-gw-${project.version}
```

Comment:
We don't need this distribution for integration tests

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/641/commits/e4cec950c82607f9dc56e753e32bfba1f514a32f.